### DISCUSSION TITLE
How to use on vanillarite server or install in technic launcher?

### DISCUSSION THREAD

**Discussion by TheFriendlyWendigo** [MrKinau/FishingBot#209] [Q&A]
I've been using the technic launcher to access a vanillarite server and would like to use the bot for afk fishing but I have no idea to use it with this launcher/client.

**Comment by MrKinau** [comment]
It's not a mod for auto fishing. It's a standalone bot, that works without the Minecraft client. It's intended to be run on an external server 24/7 without the need to keep a resource intensive Minecraft window open.
